packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -1,6 +1,14 @@
 <template>
 	<section>
-		<tera-operator-placeholder :node="node" />
+		<div
+			v-if="node.state.transientModelConfig.id && node.state.transientModelConfig.id.length > 0"
+			class="pl-2 pr-2 pb-3"
+		>
```

Comment:
Why `transientModelConfig.id` can have a length? Do you want use `isEmpty` from lodash instead?
Also with all that classes, you can just make a proper CSS selector like 
```css
div {
   padding-left: ....
  ...
  h6 {
      padding-bottom
  }

  h6 + p {
     text-subdued
  }
}
```

Reply:
Done!